static/index.html

```diff
@@ -57,6 +57,31 @@ <h1 id="grapheneos">
             href="https://attestation.app/about">about page</a> and <a
             href="https://attestation.app/tutorial">tutorial</a>. You can also extend these with
             support for your own builds.</p>
+            <h2 id="overview"><a href="#overview">Overview</a></h2>
+            <ul>
+                <li>
+                    GrapheneOS does <em>not</em> support Google play services. Some apps may not work, or only have partial functionality.
+                </li>
+                <li>
+                    GrapheneOS does <em>not</em> support rooting. This goes against its design goals and will <b>never</b> be supported.
```

Comment:
Not clear what this means and it definitely doesn't belong on the main page. It also has to be properly explained if it's written somewhere, but I'm not sure it belongs on the web site at all.